### ISSUE TITLE
Clarification on current state of `olefile` Python package

### ISSUE THREAD

**Issue by follower** [vadmium/python-altium#12]
**TL;DR:**

 * Thanks for your work on `python-altium`.

 * It seems the dependency installation situation can now be much simpler than in the past but to gain this:

    * A patch (linked below) to use `olefile` would need to be committed.
    * The `ReadMe` would need to be updated to simply instruct people to install `olefile` via `pip`.

### Context

Thanks for your work on `python-altium`.

I encountered an issue during installation/first use that I thought I'd document here in case it was helpful for others.

### Fork/version issues with `OleFileIO`/`OleFileIO_PL` package dependency  

It seems that:

 *  "OleFileIO library. Version 2.4+ of the Pillow fork" no longer exists and Pillow now uses the upstream `olefile` package. (See: [1](https://github.com/python-pillow/Pillow/blob/7855a6fc2f7a043c12f1d0c4d0c1a008e10d18a9/docs/deprecations.rst#pilolefileio), [2](https://github.com/python-pillow/Pillow/blob/c240db22283adcd68a5b3050f1dc7d4a6e44d282/docs/releasenotes/6.0.0.rst#removed-deprecated-pilolefileio), [3](https://github.com/python-pillow/Pillow/blob/fdbd719da4c77c7e23e2e9e9b71d0d177f2d3369/docs/releasenotes/5.0.0.rst#olefileiopy), [4](https://github.com/python-pillow/Pillow/blob/fdbd719da4c77c7e23e2e9e9b71d0d177f2d3369/docs/releasenotes/4.0.0.rst#olefileiopy))
 * Additionally, "version 0.30 of the OleFileIO_PL fork," *also* no longer exists and has been renamed to be `olefile`.

Unfortunately the [`OleFileIO_PL`](https://pypi.org/project/OleFileIO_PL/) package still seems to exist separately from [`olefile`](https://pypi.org/project/olefile/) but presents itself as `olefile` when installed.

With Python3 in particular this situation seems to lead to problems when first installing `olefile`, uninstalling `olefile` and then attempting to install `OleFileIO_PL` (in part due to `pip` hiding an error.).

(Further details here: <https://github.com/decalage2/olefile/issues/123>.)

### Use of renamed `olefile` dependency

Additionally, `python-altium` doesn't work out of the box with the now "standard" `olefile` but from looking at the project forks I noticed @martonmiklos has made a patch for this but hasn't yet created a PR for it:

 * <https://github.com/martonmiklos/python-altium/commit/be51f32908d589deb8c5cf8659b29b5e5e9b416b>

I haven't tested the patch yet.

### Solution

It seems the dependency installation situation can now be much simpler than in the past but to gain this:

 * The patch (linked above) to use `olefile` would need to be committed.
 * The `ReadMe` would need to be updated to simply instruct people to install `olefile` via `pip`.

*Edited:* Added Pillow-related links. Added link describing install issue.

**Comment by martonmiklos** [comment]
Hi folks, @follower  @vadmium

If you want me to submit a PR just let me know.

**Comment by follower** [comment]
Thanks for your reply & offer @martonmiklos.

While I obviously have no ability to merge a PR, if what you have is working I think having a PR would at least be helpful for people to apply themselves and/or be more likely to be merged. :)

**Comment by vadmium** [comment]
I am happy to merge commits directly, but in this case I think I can simplify the change to one line, with no other code changes needed. I pushed a new branch _olefile_ (https://github.com/vadmium/python-altium/compare/olefile), which I’ll merge when I get around to it unless anyone sees any problems.

**Comment by vadmium** [comment]
Merged the _olefile_ branch, better eleven months later than never :)
